src/library.js

```diff
@@ -49,6 +49,9 @@ LibraryManager.library = {
 
   opendir__deps: ['$FS', '__setErrNo', '$ERRNO_CODES', 'open'],
   opendir: function(dirname) {
+#if USE_PTHREADS
+    if (ENVIRONMENT_IS_PTHREAD) return _emscripten_sync_run_in_main_thread_1({{{ cDefine('EM_DEFERRED_OPENDIR') }}}, dirname);
```

Comment:
these deferred calls are proxied to the main thread? the name seems a little confusing. it's not that we defer them and forget about them, which is what "defer" says to me at least. Also, we use the term "proxying" elsewhere, so `_PROXIED` seems better?